### PR TITLE
Adds SVGO and optimizes images

### PR DESCRIPTION
**Pull request by fribmendes**

### CODE REVIEW

File: vendor/assets/stylesheets/landing_page/flexslider/fonts/flexslider-icon.svg [hunk 1]
```diff
@@ -1,19 +1 @@
...
-<glyph unicode="&#xf004;" d="M 384.00,336.00l0.00-352.00 q0.00-6.50 -4.75-11.25t-11.25-4.75l-128.00,0.00 q-6.50,0.00 -11.25,4.75t-4.75,11.25l0.00,352.00 q0.00,6.50 4.75,11.25t 11.25,4.75l 128.00,0.00 q 6.50,0.00 11.25-4.75t 4.75-11.25zM 160.00,336.00l0.00-352.00 q0.00-6.50 -4.75-11.25t-11.25-4.75l-128.00,0.00 q-6.50,0.00 -11.25,4.75t-4.75,11.25l0.00,352.00 q0.00,6.50 4.75,11.25t 11.25,4.75l 128.00,0.00 q 6.50,0.00 11.25-4.75t 4.75-11.25z" horiz-adv-x="384"  />
-<glyph unicode="&#xf005;" d="M 402.75,208.00q0.00-13.25 -9.25-22.50l-162.75-162.75q-9.50-9.50 -22.75-9.50q-13.50,0.00 -22.50,9.50l-162.75,162.75q-9.50,9.00 -9.50,22.50q0.00,13.25 9.50,22.75l 18.50,18.75q 9.75,9.25 22.75,9.25q 13.25,0.00 22.50-9.25l 121.50-121.50l 121.50,121.50q 9.25,9.25 22.50,9.25q 13.00,0.00 22.75-9.25l 18.75-18.75q 9.25-9.75 9.25-22.75z" horiz-adv-x="416"  />
-<glyph unicode="&#x20;" horiz-adv-x="224" />
-<glyph class="hidden" unicode="&#xf000;" d="M0,384L 448 -64L0 -64 z" horiz-adv-x="0" />
-</font></defs></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg"/>
```
Comment (joaofcosta) [inline vendor/assets/stylesheets/landing_page/flexslider/fonts/flexslider-icon.svg]: @frmendes Is this supposed to contain this line only? The SVG is gone right? Are we not using this specific SVG?

File: vendor/assets/stylesheets/landing_page/rs-plugin/font/revicons.svg [hunk 2]
```diff
@@ -1,54 +1 @@
...
-<glyph glyph-name="equalizer" unicode="&#xe83a;" d="m576 239l0-112l-55 0l0-167q0-23-17-40t-39-17t-39 17t-17 40l0 167l-56 0l0 112l56 0l0 503q0 24 17 39t38 16q24 0 41-16t16-39l0-503l55 0z m335 335l0-112l-55 0l0-502q0-23-16-40t-41-17q-23 0-39 17t-16 40l0 502l-56 0l0 112l56 0l0 168q0 24 16 39t39 16t41-16t16-39l0-168l55 0z m-670-112l0-111l-55 0l0-391q0-23-16-40t-40-17q-23 0-39 17t-17 40l0 391l-56 0l0 111l56 0l0 280q0 24 16 39t40 16t40-16t16-39l0-280l55 0z" horiz-adv-x="928" />
-<glyph glyph-name="layers-alt" unicode="&#xe804;" d="m18 127l446-111l447 111l0-111l-447-113l-446 113l0 111z m0 224l446-112l447 112l0-112l-447-112l-446 112l0 112z m0 223l446-112l447 112l0-112l-447-111l-446 111l0 112z m0 223l446-112l447 112l0-112l-447-111l-446 111l0 112z" horiz-adv-x="928" />
-<glyph glyph-name="popup" unicode="&#xe828;" d="m700 750q42 0 71-29t29-71l0-400q0-40-29-70t-71-30l-400 0q-40 0-70 30t-30 70l0 402q0 40 29 69t71 29l400 0z m0-500l0 400l-400 0l0-400l400 0z m-600 100l0-300l300 0l0-100l-300 0q-40 0-70 30t-30 70l0 300l100 0z" horiz-adv-x="800" />
-</font>
-</defs>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg"/>
```
Comment (joaofcosta) [inline vendor/assets/stylesheets/landing_page/rs-plugin/font/revicons.svg]: @frmendes Is this supposed to contain this line only? The SVG is gone right? Are we not using this specific SVG?

File: vendor/assets/stylesheets/landing_page/flexslider/fonts/flexslider-icon.svg (hunk 1 above)
Comment (frmendes) [inline vendor/assets/stylesheets/landing_page/flexslider/fonts/flexslider-icon.svg]: We are probably not using this specific SVG but I'll re-add it unoptimized.

**Comment by frmendes** [comment]
@joaofcosta ping

**Pull request merged**